### PR TITLE
ch05 - editor comments

### PR DESCRIPTION
**Pull request by ismaelcv**

### CODE REVIEW

File: chapter05/docker-compose.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@ services:
     <<: *airflow-common
     command: webserver
     ports:
-      - "8080:8080"
```
Comment (krisgeus) [inline chapter05/docker-compose.yml:123]: Can you undo this change to keep it consistent with the other chapters?

File: chapter05/dags/13_dag_decorator.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+from airflow.decorators import task, dag
+
+@dag( 
+    dag_id="13_dag_decorator",
+    start_date=pendulum.today("UTC").add(days=-3),
+    schedule="@daily",
+)
+def taskflow_api_decorator(): 
+
+    @task #C
```
Comment (danielvdende) [inline chapter05/dags/13_dag_decorator.py:17]: `#C`?

**Review by danielvdende — COMMENTED** [review]
1 minor comment.

**Review by krisgeus — APPROVED** [review]
